source/client-side-encryption/tests/README.rst

```diff
@@ -158,6 +158,14 @@ Each YAML file has the following keys:
   - ``outcome``: |txn|
 
 
+Credentials
+===========
+
+Test credentials are available in AWS Secrets Manager. See https://wiki.corp.mongodb.com/display/DRIVERS/Using+AWS+Secrets+Manager+to+Store+Testing+Secrets for more background on how the secrets are managed.
+
+Test credentials to KMS are located in "drivers/client-side-encryption".
+
+Test credentials to create environments are available in "drivers/gcpkms" and "drivers/azurekms".
```

Comment:
We already have these credentials in the existing "drivers/csfle" vault. Should we remove these other vaults in favor of "drivers/csfle"?

Reply:
CC: @blink1073

Reply:
Note: I used `drivers/csfle` in https://github.com/mongodb-labs/drivers-evergreen-tools/pull/390

Reply:
Updated to use `drivers/csfle` and scheduled `drivers/client-side-encryption` for deletion. Thank you for adding the secrets. 

> Should we remove these other vaults in favor of "drivers/csfle"?

I propose keeping `drivers/gcpkms` and `drivers/azurekms` separate. Those vaults are in use by [Python](https://github.com/mongodb/mongo-python-driver/blob/85b0c0e3c13cbb7c005147a95292185ca1fc4b1d/.evergreen/config.yml#L988) and [Go](https://github.com/mongodb/mongo-go-driver/blob/1fd1bc2978d1aa163e39258334e9421d1464da23/.evergreen/config.yml#L2415). I expect the Azure and GCP prose tests are run in a separate task group to manage the lifetime of the created VM. I do not see much value in combining the credentials.

Reply:
I agree with keeping them separate